src/operator-docs/transformation/window.ts

```diff
@@ -1,6 +1,68 @@
 import { OperatorDoc } from '../operator.model';
 
 export const windowOperator: OperatorDoc = {
-  'name': 'window',
-  'operatorType': 'transformation'
+  name: 'window',
+  operatorType: 'transformation',
+  signature:
+    'public window(windowBoundaries: Observable<any>): Observable<Observable<T>>',
```

Comment:
We can remove the generic signatures here, can be done later in one go though. Don't think it's worth holding up the PR.